### PR TITLE
A limited local run

### PR DESCRIPTION
**Pull request by janus**
How to start up the server and run the app locally.

### CODE REVIEW

**Comment by janus** [comment]
@owocki 
This PR is from @janus . You asked me via slack to do this.

File: docs/RUNNING_LOCALLY_LIIMITED_EDITION.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# How to run gitcoin web locally on your computer
```
Comment (owocki) [inline docs/RUNNING_LOCALLY_LIIMITED_EDITION.md]: why the new file? just edit the old file

**Comment by thelostone-mc** [comment]
@janus is this a duplicate of https://github.com/gitcoinco/web/pull/5546

If so can we close this out ?  ^_^

**Comment by janus** [comment]
Close . Duplicate

**Pull request closed without merging**